docs/misc/dvd-remux.md

```diff
@@ -0,0 +1,670 @@
+For a basic episode remux,
+you'll typically want to focus on the main title and chapters.
+However, for a complete preservation,
+you might want to capture all available angles as well.
+
+??? question "What are PGCs?"
+    PGCs (Program Chains) are part of the DVD video structure hierarchy.
+    At the lowest level, multiple VOBUs (Video Object Units) form a cell.
+    One or more cells make up a program,
+    and one or more programs form a PGC.
+    Programs can be identified as PTTs (Parts of Title),
+    also commonly known as chapters.
+    A title on a DVD consists of one or more PGCs,
+    while a menu is made up of a single PGC[^pgc-references].
+
+[^pgc-references]: For more information,
+    see [this page](https://dvd.sourceforge.net/dvdinfo/pgc.html),
+    [this page](http://www.mpucoder.com/DVD/),
+    and [this page](https://en.wikibooks.org/wiki/Inside_DVD-Video/MPEG_Format#VOBU,_Cell,_Program,_PGC).
+
+## Remuxing Methods
+
+Below are multiple methods to remux your DVDISO.
+If you are unsure if one method
+produced the correct result,
+try another method
+and see if the result
+is the same.
+
+!!! example "Remuxing methods"
+
+    === "FFmpeg"
+
+        !!! warning "Version"
+            This method requires BtbN nightly FFmpeg build!
+            If you are using an older version,
+            you will need to upgrade.
+            You can find the latest BtbN nightly builds [here](https://github.com/BtbN/FFmpeg-Builds/releases/download/latest/ffmpeg-n7.1-latest-win64-gpl-7.1.zip).
+
+        !!! info "Documentation"
+            For more information on the command line arguments,
+            see the [FFmpeg documentation](https://ffmpeg.org/ffmpeg-formats.html#dvdvideo).
+
+        FFmpeg can be used to help split a DVDISO by Title, Angle, or Chapters.
+
+        You will ideally want all of the above,
+        and the most straight-forward way to find them
+        is by using [MPC-HC](https://github.com/clsid2/mpc-hc)
+        (note that VLC and mpv
+        do not support all the functionality
+        covered in this guide
+        as of the time of writing).
+
+        ??? question "What _do_ VLC and mpv support?"
+            On Linux,
+            mpv has okay-ish libdvdnav support,
+            meaning you can use scripts
+            such as [mpv-dvd-browser](https://github.com/CogentRedTester/mpv-dvd-browser)
+            to navigate to different titles.
+            Angles are still only accessible via the command line,
+            and menus are unsupported.
+            VLC also supports most functionalities,
+            but has no support for different angles.
+
+            If you don't need angle/menu support,
+            and _really_ don't want to use MPC-HC,
+            either of the above options
+            may work for you.
+            Nonetheless,
+            this guide
+            will only cover MPC-HC usage.
+
+        !!! example "How to view your DVD"
```

Comment:
It'd make more sense to have this section above the program-specific part, since it's not really specific to remuxing with ffmpeg.